lib/importer.js

```diff
@@ -99,6 +99,13 @@ var loadAndStoreDocs = function(db, couchdb, concurrentDocLimit, docsToDownload,
         INSERT_DOC_STMT,
         postgresTable,
         couchDbResult.rows.map(function(row) {
+          const isUserDoc = row.doc && row.doc.type === 'user' && row.doc._id.startsWith('org.couchdb.user:');
+          if (isUserDoc) {
+            delete row.doc.password_scheme;
+            delete row.doc.derived_key;
+            delete row.doc.salt;
+          }
+
```

Comment:
I lightly would suggest parallel structure to our `sanitise()` call. That'd be add a function, maybe `removePasswordHash()` or something, that we'd call here instead of doing the declaration of `isUserDoc` and the `delete` inline within the `map()` call. 

The main reason is that `derived_key` doesn't pop out why you'd delete it, but `removePasswordHash` makes a lot sense to be here!

Reply:
In addition it's better not to define a function in a loop. Depending on the compiler it's bad for performance, but if nothing else it makes reuse somewhere else harder.

Furthermore this is a complicated line to put as an argument to a function - better to create a new variable called `docs` to store the `map` result and then invoke the `format` function.

```js
// at the top of the file
      const isUserDoc = doc => doc && doc.type === 'user' && doc._id.startsWith('org.couchdb.user:');

// inline here 
      const docs = couchDbResult.rows.map(row => {
        if (isUserDoc(row.doc)) {
          delete row.doc.password_scheme;
          delete row.doc.derived_key;
          delete row.doc.salt;
        }
        return [row.doc];
      });
      var insertSql = format(INSERT_DOC_STMT, postgresTable, docs);
```

Reply:
Thanks guys. I believe I addressed all feedback here. I'm not quite 100% sure... sorry if I missed something. 
Defs more readable now.